deploy/editableazuredeploy.json

```diff
@@ -135,7 +183,16 @@
             "VV_COLOR_PALETTE": "[parameters('colorPalette')]",
             "VV_WAITING_TITLE": "[parameters('waitingTitle')]",
             "VV_WAITING_SUBTITLE": "[parameters('waitingSubtitle')]",
-            "VV_LOGO_URL": "[parameters('logoUrl')]"
+            "VV_LOGO_URL": "[parameters('logoUrl')]",
+            "VV_POSTCALL_SURVEY_TYPE": "[parameters('postCallSurveyType')]",
+            "VV_POSTCALL_SURVEY_OPTIONS_SURVEYURL": "[parameters('postCallSurveyOptionsSurveyUrl')]",
+            "VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_TITLE": "[parameters('postCallSurveyOneQuestionPollTitle')]",
+            "VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_PROMPT": "[parameters('postCallSurveyOneQuestionPollPrompt')]",
+            "VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_TYPE": "[parameters('postCallSurveyOneQuestionPollType')]",
+            "VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_ANSWER_PLACEHOLDER": "[parameters('postCallSurveyOneQuestionPollAnswerPlaceholder')]",
+            "VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_SAVE_BUTTON_TEXT": "[parameters('postCallSurveyOneQuestionPollSaveButtonText')]",
+            "VV_COSMOSDB_ENDPOINT": "[if(equals(parameters('postCallSurveyType'),'onequestionpoll'), reference(resourceId('Microsoft.DocumentDB/databaseAccounts', parameters('cosmosDbAccountName'))).documentEndpoint, '')]",
+            "VV_COSMOSDB_NAME": "[parameters('cosmosDbName')]"
```

Comment:
This should be `VV_COSMOS_DB_NAME` `VV_COSMOS_DB_ENDPOINT`. Also, these Cosmos env vars should be added in all 3 ARM templates